nixos/modules/services/mail/protonmail-bridge.nix

```diff
@@ -0,0 +1,42 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.protonmail-bridge;
+in
+{
+
```

Comment:
Please pass this new files through `nixfmt-rfc-style`. Remove trailing blank lines too.

Reply:
The file was already formatted though, are there any syntax conventions you guys follow that's not enforced by the formatter? e.g. trailing blank lines